templates/openshift-scc.yaml

```diff
@@ -112,7 +112,7 @@ requiredDropCapabilities:
 runAsUser:
   type: RunAsAny
 seLinuxContext:
-  type: MustRunAs
+  type: RunAsAny
```

Comment:
1. RunAsAny alleviates the point of failure of validating the available seLinuxOptions which are not set properly in the postgresql in openshift.
2. Here is definition of the current and proposed type of SELinuxContext.
SELinuxContext strategy:
MustRunAs - Requires seLinuxOptions to be configured if not using pre-allocated values. Uses seLinuxOptions as the default. Validates against seLinuxOptions.
RunAsAny - No default provided. Allows any seLinuxOptions to be specified.

Reply:
If not okay to use `RunAsAny`, the changes in the `values.yaml` would be required to pass validation of `seLinuxOptions` with `MustRunAs`.